### ISSUE TITLE
advanced search issue

### ISSUE THREAD

**Issue by nandotech** [SnosMe/awakened-poe-trade#32]
seems getting a bad api response. For this armor below, happens every time, but on the bow I can only get it if I choose to ignore _ALL_ the modifiers (aside from base/links/ilvl/shaper etc)

When attempting to filter for only specific stats:

![image](https://user-images.githubusercontent.com/2347606/73274979-7d47d600-41b4-11ea-8b0a-3da5d254c9e9.png)

item:
```
Rarity: Rare
Rune Mantle
Loricated Ringmail
--------
Armour: 699 (augmented)
Energy Shield: 155 (augmented)
--------
Requirements:
Level: 58
Str: 84
Int: 84
--------
Sockets: G-R-R-R-R-G 
--------
Item Level: 84
--------
+23 to Strength
+41 to Armour
91% increased Armour and Energy Shield
+18 to maximum Energy Shield
+105 to maximum Life
+24% to Fire Resistance
+29% to Cold Resistance
--------
Corrupted
```

bow:

```
Rarity: Rare
Brood Siege
Decimation Bow
--------
Bow
Quality: +28% (augmented)
Physical Damage: 54-142 (augmented)
Elemental Damage: 68-122 (augmented)
Critical Strike Chance: 6.95% (augmented)
Attacks per Second: 1.20
--------
Requirements:
Level: 60
Dex: 170 (unmet)
--------
Sockets: G-G-G-G-B G 
--------
Item Level: 84
--------
39% increased Critical Strike Chance (implicit)
--------
+1 to Level of Socketed Gems
47% increased Physical Damage
Adds 68 to 122 Fire Damage
4% increased Movement Speed
--------
Shaper Item
```

**Comment by nandotech** [comment]
Also...separate but related idk if you want to open another issue.

I clicked the link to open official PoE trade on that armor....it was _NOT_  filtering for ANY of the stats. All the stats were actually listed, but none were "checked". When actually checking them I realize there is NO OTHER armor on the market with these specific mods.

They seem to be less important mods, armor for example, and the overall price estimate at top isn't terribly far off but definitely on the low end I think.

Anyway, point is as QOL would be better to have the stats enabled by default? Would even make the list directly below mods more relevant maybe?

Last thing-

* When searching a non unique item, also add the "Any Non-Unique" to search so it doesn't include random 6 link uniques for example.

Thanks again for all your work!

**Comment by SnosMe** [comment]
Indeed, this is a bug in the absence of search results (and was immediately found by one of the people in the discord after the release).

> it was NOT filtering for ANY of the stats
At this point in the investigation, you were mistaken.

> QOL would be better to have the stats enabled by default
Yes, it will be so. But first need to implement pseudo mods, otherwise there will always be ~ 0 results.

**Comment by nandotech** [comment]
Well I mean...when you say I'm mistaken...for example that chest piece...

I don't mean that stats aren't being taken into the price *estimate*, but rather not being accounted for in the list of prices directly below.

The price list below begins at like 15c (and there's like the entire list full) but the price prediction tells me 39-50c. (See screenshot)

So to investigate...I click the pop-out to search results...and I'm seeing tons of results _NOT_ seraching for the mods (aside from links, ilvl, corrupted) the prices coincide.

If I literally just click the stat filters I can see they are all not checked...hence the price list is clearly not filtering for any of the STATS (and not the "top level" modifiers).

Obviously if it WERE filtering it would actually get zero results anyway. Honestly for that particulary piece, even if you change _EVERYTHING_ to pseudo and even remove the number values, you still won't get any results (aside from the one I have posted until you remove item level). Not that it's particularly good, just random combination of stats and 6 link corrupt.

**Comment by SnosMe** [comment]
1. Was fixed in 0.6.1 `Correctly handle the "No Results" state`

2. > QOL would be better to have the stats enabled by default

More community discussions are welcome.
